BatteryStatus.cs

```diff
@@ -0,0 +1,20 @@
+using System;
+
+namespace Checker
+{
+    class BatteryStatus
```

Comment:
Change the class name

Reply:
class name has been changed to BatteryManagementSystem